.github/scripts/status_update_common.py

```
@@ -0,0 +1,226 @@
+# Projects status 更新の共通処理
+
+from math import e
```

Ruff (0.8.2)

`math.e` imported but unused

Remove unused import: `math.e`

(F401)